packages/fxa-content-server/app/styles/tailwind.css

```diff
@@ -1,11 +1,12 @@
-/* When content-server is using Tailwind exclusively use "@tailwind" directives instead of
- * imports. See: https://tailwindcss.com/docs/using-with-preprocessors */
```

Comment:
I removed this comment since I was incorrect here, we actually have to use `tailwindcss/base` etc. when we're using PostCSS. Updated L8 to match other CSS imports.